test/input/2.0.0/gitter-streaming.yml

```diff
@@ -0,0 +1,137 @@
+asyncapi: 2.0.0
+info:
+  title: Gitter Streaming API
```

Comment:
Why adding this for `2.0.0` as we already have it present in `1.2.0`. No need to duplicate it I think? 🤔

Reply:
Removed.